numpy/core/fromnumeric.py

```diff
@@ -649,6 +649,14 @@ def transpose(a, axes=None):
     >>> np.transpose(x, (1, 0, 2)).shape
     (2, 1, 3)
 
+    >>> x = np.arange(120).reshape(2, 3, 4, 5)
+    >>> x.shape
+    (2, 3, 4, 5)
+
+    >>> x_transposed = np.transpose(x)
+    >>> x_transposed.shape
+    (5, 4, 3, 2)
```

Comment:
For consistency with the example above, I'd combine these lines:

```suggestion
    >>> np.transpose(x).shape
    (5, 4, 3, 2)
```